.github/workflows/draft_release.yml

```diff
@@ -0,0 +1,207 @@
+name: Publish draft release
+
+on:
+  workflow_dispatch:
+    inputs:
+      ref1:
+        description: The 'from' tag to use for the diff
+        default: statemine-v5.0.0
+        required: true
+      ref2:
+        description: The 'to' tag to use for the diff
+        default: HEAD
+        required: true
+      pre_release:
+        description: For pre-releases
+        default: "true"
+        required: true
+
+jobs:
+  get-rust-versions:
+    runs-on: ubuntu-latest
+    container:
+      image: paritytech/ci-linux:production
+    outputs:
+      rustc-stable: ${{ steps.get-rust-versions.outputs.stable }}
+      rustc-nightly: ${{ steps.get-rust-versions.outputs.nightly }}
+    steps:
+      - id: get-rust-versions
+        run: |
+          echo "::set-output name=stable::$(rustc +stable --version)"
+          echo "::set-output name=nightly::$(rustc +nightly --version)"
+
+  build-runtimes:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        runtime: ["shell", "statemine", "statemint", "westmint", "rococo"]
```

Comment:
I don't see the need for building the shell and rococo chains by default. We're not going to be releasing them again any time soon.

Reply:
Those are built in parallel so there is no time penalty.
They will NOT but showing up in the release notes thanks to [those lines](https://github.com/paritytech/cumulus/pull/761/files#diff-cf9ff1c5bc30168737462f98fc8932addf46407f697d9914bd8e57ffb61da05eR124-R125).

Moreover, I cannot simply remove those from here as those rutime data are expected [here](https://github.com/paritytech/cumulus/pull/761/files?authenticity_token=wBLIighCUp8nkRS0I92i%2FzcCPbbTmlcbYgQTdFJ6MC2V4rc3AHwGhAlqbniQhkA78KocNe%2F6kKqGIQa%2F1V1WEg%3D%3D&file-filters%5B%5D=.lock&file-filters%5B%5D=.md&file-filters%5B%5D=.rb&file-filters%5B%5D=.sh&file-filters%5B%5D=.tera&file-filters%5B%5D=No+extension&file-filters%5B%5D=dotfile&hide-deleted-files=true#diff-5c9e7b4c43db13e41f91b3d40d328875c6e87fa6d06c55641347756f1b5fe1d9R87-R112).

We can improve later I think.

Reply:
There is also a benefit, they will be built and attached which makes it easy for us when we need them.